noobaa/noobaa-core
# system address form

### Explain the changes:
- system address form

### Issues: Fixed / Gap #xxx
- 

### Testing Instructions:
1. go to management
2. expand 'system-address'
3. select 'Use DNS name'
4. type address test.com
5. dirty mark is shown near the header
6. click Apply Changes
7. can't save until async validation completed

### Reminders:
- Create a new test - the first is the hardest
- User Experience is top priority
- Design for failure
- Keep it simple
- `npm test`
- Imagine a support call - Add diagnostics info, phonehome info, activity log events, external syslog
- Upgrade - DB schema, server platform, agents install, npm packages
